src/ebonite/core/objects/core.py

```diff
@@ -582,11 +590,65 @@ def __init__(self, name: str, id: str = None,
         self.params = params or {}
 
     @property
+    @_with_meta
     def model(self) -> Model:
-        raise AttributeError('Can\'t access model of unbound image')
+        m = self._meta.get_model_by_id(self.model_id)
+        if m is None:
+            raise errors.NonExistingModelError(self.model_id)
+        return m
 
     @model.setter
     def model(self, model: Model):
         if not isinstance(model, Model):
             raise ValueError('{} is not Model'.format(model))
         self.model_id = model.id
+
+
+class RuntimeEnvironment(EboniteObject):
+    def __init__(self, name: str, id: str = None,
+                 host: str = None, port: int = None,
+                 author: str = None, creation_date: datetime.datetime = None):
+        super().__init__(id, name, author, creation_date)
+        self.host = host
+        self.port = port
+
+    def get_uri(self) -> str:
+        return f'{self.host}:{self.port}' if self.host else ''
+
+
+class RuntimeInstance(EboniteObject):
+    def __init__(self, name: str, id: str = None,
+                 image_id: str = None, environment_id: str = None, params: Dict[str, Any] = None,
+                 author: str = None, creation_date: datetime.datetime = None):
+        super().__init__(id, name, author, creation_date)
+        self.image_id = image_id
+        self.environment_id = environment_id
+        self.params = params or {}
```

Comment:
Here, in `Image` and `RuntimeEvinronment`, I suggest we use pyjackson for params field.
I mean it like this: params field has type `ImageParams` (or `RuntimeEvinronmentParms` etc), which has `@type_field` pyjackson decorator.
`DockerImage` from docker_objects is a child of `ImageParams` and so on. This way you don't need methods like `to_core_image`

Reply:
I propose to change it after #51 because of expected merging problems